stylesheets/scss/_colors.scss

```diff
@@ -7,11 +7,11 @@ $availability-link-all:#5aa47f;
 
 $color-mint-green-shadow: #004d3b;
 $color-mint-green-hover: #00a982;
-$color-disabled-grey: #b1b1ad;
-$color-disabled-hover-grey: #a7a7a3;
-$color-disabled-shadow: #656560;
-$color-disabled-shadow-alternative: #8c8c86;
-$color-disabled-grey-dark: #444444;
+$color-button-grey: #b1b1ad;
+$color-hover-grey: #a7a7a3;
+$color-shadow: #656560;
+$color-shadow-alternative: #8c8c86;
+$color-grey-dark: #444444;
```

Comment:
The reason is that the colors aren't for disabled buttons, as I first thought. I changed the variables to prevent confusion from using colors, that has nothing to do with anything being disabled.